sdks/java/io/sns/src/main/java/org/apache/beam/sdk/io/sns/AwsClientsProvider.java

```diff
@@ -0,0 +1,17 @@
+package org.apache.beam.sdk.io.sns;
```

Comment:
Missing Apache headers in this and other files (that's the reason the build broke)

Reply:
Added Apache headers to all files.